### PR TITLE
Allow sql cli to print more types

### PR DESCRIPTION
**Pull request by tamird**
Closes #1961.

### CODE REVIEW

**Comment by tamird** [comment]
Turns out this wheel was already invented in the standard library.

File: sql/logic_test.go [hunk 1]
```diff
@@ -135,43 +134,12 @@ type logicQuery struct {
...
-}
-
 // logicTest executes the test cases specified in a file. The file format is
 // taken from the sqllogictest tool
 // (http://www.sqlite.org/sqllogictest/doc/trunk/about.wiki) with various
 // extensions to allow specifying errors and additional options. See
 // https://github.com/gregrahn/sqllogictest/ for a github mirror of the
 // sqllogictest source.
-//
-// TODO(pmattis): We currently cannot run the tests from sqllogictest due to
```
Comment (MBerhault) [inline sql/logic_test.go]: this todo can probably stay

File: cli/sql.go [hunk 2]
```diff
@@ -82,16 +82,23 @@ func processOneLine(db *sql.DB, line string) error {
 	// Format all rows using tabwriter.
 	tw := tabwriter.NewWriter(os.Stdout, 0, 8, 0, '\t', 0)
 	fmt.Fprintf(tw, "%s\n", strings.Join(cols, "\t"))
-	strs := make([]string, len(cols))
```
Comment (MBerhault) [inline cli/sql.go]: can you add a todo (fine to assign it to me) to add a test for the cmdline interface?

**Comment by MBerhault** [comment]
LGTM

File: sql/logic_test.go (hunk 1 above)
Comment (tamird) [inline sql/logic_test.go]: why? https://twitter.com/cockroachdb

Comment (MBerhault) [inline sql/logic_test.go]: oh. nevermind then.

File: cli/sql.go (hunk 2 above)
Comment (tamird) [inline cli/sql.go]: filed an issue

File: sql/logic_test.go [hunk 3]
```diff
@@ -135,43 +134,12 @@ type logicQuery struct {
...
-// insufficient SQL coverage (e.g. lack of subqueries and aggregation
-// functions). We should work towards fixing that.
```
Comment (vivekmenezes) [inline sql/logic_test.go]: just checking that removing this TODO is not related. You just happened to remove it right.

File: sql/logic_test.go [hunk 4]
```diff
@@ -408,7 +375,7 @@ func (t *logicTest) execQuery(query logicQuery) {
 	}
 	vals := make([]interface{}, len(cols))
```
Comment (vivekmenezes) [inline sql/logic_test.go]: can you just make a slice of NullString?

Comment (vivekmenezes) [inline sql/logic_test.go]: And same for the one in cli/sql.go

**Comment by vivekmenezes** [comment]
Nice!

LGTM

File: sql/logic_test.go (hunk 3 above)
Comment (tamird) [inline sql/logic_test.go]: Correct. The removal is in a separate commit.

File: sql/logic_test.go (hunk 4 above)
Comment (tamird) [inline sql/logic_test.go]: unfortunately not. [`Scan`](http://godoc.org/database/sql#Row.Scan)'s signature is `func (r *Row) Scan(dest ...interface{}) error`, and go does not coerce in this case.
